workos/__init__.py

```diff
@@ -1,9 +1,16 @@
 import os
 
 from workos.__about__ import __version__
-from workos.client import client
+from workos.client import SyncClient
+from workos.async_client import AsyncClient
 
 api_key = os.getenv("WORKOS_API_KEY")
 client_id = os.getenv("WORKOS_CLIENT_ID")
-base_api_url = "https://api.workos.com/"
-request_timeout = 25
+base_api_url = os.getenv("WORKOS_BASE_URL", "https://api.workos.com/")
+request_timeout = int(os.getenv("WORKOS_REQUEST_TIMEOUT", "25"))
+
+
+client = SyncClient(base_url=base_api_url, version=__version__, timeout=request_timeout)
+async_client = AsyncClient(
+    base_url=base_api_url, version=__version__, timeout=request_timeout
+)
```

Comment:
To prevent circular imports and promote client flexibility (such as for when we add FGA APIs), I pass in the base API URL. This means the base URL can't be updated on the fly with `workos.base_api_url = "new url"` like we do currently, so instead I'm pulling from an environment variable.

This feels pretty standard to me, but any thoughts?

Reply:
That seems OK. Could a developer just create an instance of the client themselves if they didn't want to use environment variables?

Reply:
That's a good point! They definitely could.